src/backend/dx11/src/device.rs

```diff
@@ -1889,7 +1889,18 @@ impl device::Device<Backend> for Device {
 
         for binding in bindings.iter() {
             let content = DescriptorContent::from(binding.ty);
-            total.add_content_many(content, binding.stage_flags, binding.count as _);
+            // If this binding is used by the graphics pipeline and is a UAV, it belongs to the "Output Merger"
+            // stage, so we only put them in the fragment stage to save redundant descriptor allocations.
+            let stage_flags =
```

Comment:
could be simpler to just write this as
```rust
if content.contains(DescriptorContent::UAV) {
  stage_flags &= !pso::ShaderStageFlags::VERTEX;
}
```
looks to be equivalent to what you wrote?

Reply:
~~This does seem equivalent if you only consider [VS, PS, COMP] as your shader set, but the dx11 backend could very well support tess/geo, so I'm trying to include them whenever I write logic like this.~~

This behaves differently if _only_ vertex is set; it will just remove VERTEX without also setting FRAGMENT.

Musing over if there's a way to simplify this...